framework/base/variables/_font-scale.scss

```diff
@@ -2,11 +2,15 @@
 
 $font-size: (
   none: 0,
-  xs: 0.75rem,
-  s: 0.875rem,
+  xxs: 0.67rem,
+  xs: 0.78rem,
+  s: 0.89rem,
   m: 1rem,
-  l: 1.25rem,
-  xl: 1.5rem,
-  xxl: 2.25rem,
-  xxxl: 3rem
+  l: 1.22rem,
+  xl: 1.67rem,
+  xxl: 2.22rem
 ) !default;
+
+:root {
```

Comment:
Should be part of the reset (`:root` is like `html` but has a hight specificity) or even before the reset